prometheus-to-sd/main.go

```diff
@@ -78,6 +80,10 @@ func main() {
 	}
 	glog.Infof("GCE config: %+v", gceConf)
 
+	go func() {
+		glog.Errorf(http.ListenAndServe(":6060", nil).Error())
```

Comment:
You can probably do it in a following way:

`glog.Error(http.ListenAndServe(":6060", nil))`